draft-bbs-signatures.md

```diff
@@ -748,6 +757,77 @@ Procedure:
 5. return (scalar_1, ..., scalar_n)
 ```
 
+### OctetsToSignature
+
+This operation describes how to decode an octet string, validate it and return the underlying components that make up the signature.
+
+```
+(A, e, s) = octets_to_signature(signature_octets)
+
+Inputs:
+
+- signature_octets (REQUIRED), octet string.
+
+Parameters:
+
+- octet_point_length, the length in bytes of the octet string output by the point_to_octets function
```

Comment:
Do we need to redefine it when its available at the cipher suite definition level? I think we need to figure out the scoping otherwise there would be loads of parameters in every definition?